pySMART/device.py

```diff
@@ -1028,6 +1034,13 @@ class members, including the SMART attribute table and self-test log.
                     for line in _stdout.split('\n'):
                         if 'power on time' in line:
                             self.diags['Power_On_Hours'] = line.split(':')[1].split(' ')[1]
+        # map temperature
+        if self.temperature is None:
```

Comment:
This is actually intended since there could be a case where temperature = 0, and `not 0` is `True` in python. Besides, it can never be assigned an empty string because of the int conversion above and I *only* want it to be assigned here if it was not already assigned via the attributes above.